surveys/2021-annual-survey/questions.md

```diff
@@ -713,6 +713,7 @@ Select one:
 - Doesn't add a feature I want
 - Stagnation
 - Instability of the language
+- Development of the language not scalable/sustainable
```

Comment:
Is this trying to capture the worry that the Rust project itself implodes or fails to keep up with the demands? I like this, but I'm not sure about the wording though I can't come up with a good alternative.